Hover text for flags on README page

akashprasher opened the pull request:
The language name will appear when we hover the flag. All results are based on Google Research. The further contribution may be required to rectify.

akashprasher committed:
Adding text="" and alt="" atttribute

akashprasher committed:
Added Title of language upto Russia

akashprasher committed:
fixing lang-flag नेपाली

akashprasher committed:
Added Values in title and alt for flags/languages

Roshanjossey commented:
Hi @akashprasher, thank you for working on this. I found a few mistakes while going through the names. Could you please check your changes with the names we have in https://github.com/firstcontributions/first-contributions/blob/master/translations/Translations.md?

akashprasher commented:
I'll make appropriate changes.

akashprasher committed:
Updating upto Igbo (Nigeria)

akashprasher committed:
Updating the Language name with respect to Flag

Updating name according to link -> https://github.com/firstcontributions/first-contributions/blob/master/translations/Translations.md

akashprasher commented:
I've made the changes, according to the list given at https://github.com/firstcontributions/first-contributions/blob/master/translations/Translations.md **Please review it.**

Thanks

akashprasher commented:
I've also deleted the line

` <kbd>[<img title="Kiswahili (Kenya)" alt="Kiswahili (Kenya)" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ke.svg" width="22">](translations/README.kws.md)</kbd> `

because it was repeating.

Roshanjossey commented:
@akashprasher, awesome. Thank you for making these changes.

akashprasher commented:
@Roshanjossey is it possible for you to add label "hacktoberfest-accepted" in this PR?

Roshanjossey commented:
Sure
